electronics_lib/Imu_Lsm6ds3trc.py

```diff
@@ -0,0 +1,71 @@
+from electronics_abstract_parts import *
+from .JlcPart import JlcPart
+
+
+class Imu_Lsm6ds3trc_Device(DiscreteChip, FootprintBlock, JlcPart):
+    def __init__(self) -> None:
+        super().__init__()
+        self.vdd = self.Port(VoltageSink(
+            voltage_limits=(1.71, 3.6)*Volt,
+            current_draw=(3*uAmp, 0.9*mAmp)  # typical values for low-power and high-performance modes
+        ))
+        self.vddio = self.Port(VoltageSink(
+            voltage_limits=(1.62*Volt, self.vdd.voltage_limits.upper()+0.1)
+        ))
+        self.gnd = self.Port(Ground())
+
+        # TODO figure out how to initialize pins
+        din_model = DigitalBidir.from_supply(
+            self.gnd, self.vddio,
+            voltage_limit_abs=(-0.3*Volt, self.vddio.voltage_limits.upper()+0.3),
+            # datasheet states abs volt to be [0.3, VDD_IO+0.3], likely a typo
+            current_limits=(0, 4)*mAmp,
```

Comment:
```suggestion
            current_limits=(-4, 4)*mAmp,
```
I think this is symmetric sink and sourcing current?

See datasheet note 2 on pg 24
> 4 mA is the maximum driving capability, i.e. the maximum DC current that can be sourced/sunk by the digital pad in order to guarantee the correct digital output voltage levels VOH and VOL